### PR TITLE
Add Python CI workflow

### PR DESCRIPTION
**Pull request by charris-msft**
This adds a new python CI workflow that does linting with flake8.

There are also placeholders in the file for installing mysql, and running pytest tests.

### CODE REVIEW

**Comment by charris-msft** [comment]
@xwen11 - would you mind reviewing?

File: contoso-supermarket/src/app.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-from importlib import import_module
```
Comment (xwen11) [inline contoso-supermarket/src/app.py:2]: Is this change intentional?

File: .github/workflows/python.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```
Comment (xwen11) [inline .github/workflows/python.yaml:1]: Love this! I see ptest part is commented. I don't have any experience on ptest. It will be cool if we add the real test cases there. CI workflow definitely should include running unit tests.

**Review by xwen11 — APPROVED** [review]
LGTM!

File: contoso-supermarket/src/app.py (hunk 1 above)
Comment (charris-msft) [reply contoso-supermarket/src/app.py:2]: Yes - the linter, flake8, found that issue so I fixed it. :)
